LICENSE.md

```diff
@@ -0,0 +1,79 @@
+<h1 align="center">Creative Commons CC0 1.0 Universal</h1>
```

Comment:
As you already noted, formatting the license this way breaks compatibility with GitHubs license recognition.
While I think that your version is more readable, I think having GitHub integration is more important.

I don't think that a lot of people actually want to read the legal jargon and GitHub shows a useful quick license overview when it recognizes the license.

I'd therefore vote against reformatting the license.